### DISCUSSION TITLE
Installing Truffle on Ububtu 22.04

### DISCUSSION THREAD

**Discussion by rhythmshandlya** [ConsenSys-archive/truffle#5085] [Installation]
## Issue

Unable to install truffle on ubuntu LTS 22.04 globally

## Steps to Reproduce

sudo npm install -g truffle

## Expected Behavior

Truffle should be installed globally

## Actual Results

npm ERR! code 1
npm ERR! path /usr/local/lib/node_modules/truffle/node_modules/ganache/node_modules/@trufflesuite/bigint-buffer
npm ERR! command failed
npm ERR! command sh -c node-gyp rebuild

## Environment
* Operating System: ubuntu LTS 22.04
* Ethereum client:
* Truffle version (`truffle version`): 5.5.6
* node version (`node --version`): 16.14.0
* npm version (`npm --version`):  8.3.1

**Comment by liarco** [comment]
Hi, I just got here by chance, but maybe this video can be helpful: https://youtu.be/imuqi6Vg3Zw (there is a section dedicated to Truffle, you might just need that part).

I hope this helps.

**Comment by gnidan** [comment]
Hey, thanks for reporting this! I converted this to a GitHub Discussion, since it's not a problem with Truffle, per se.

Could you please see [these instructions on the Infura Community Discourse](https://community.infura.io/t/installing-truffle-on-ubuntu/5113) and let us know if that helps you install Truffle?

**Comment by apdan510** [comment]
@rhythmshandlya are you still having this issue? I can assist.
